cmd/ketch/app_export_test.go

```diff
@@ -31,23 +28,22 @@ func Test_newAppExportCmd(t *testing.T) {
 	}{
 		{
 			name: "happy path",
-			args: []string{"ketch", "foo-bar", "-d", "/tmp/app"},
-			appExport: func(ctx context.Context, cfg config, chartNew chartNewFn, options appExportOptions, out io.Writer) error {
+			args: []string{"ketch", "foo-bar"},
+			appExport: func(ctx context.Context, cfg config, options appExportOptions) error {
 				require.Equal(t, "foo-bar", options.appName)
-				require.Equal(t, "/tmp/app", options.directory)
 				return nil
 			},
 		},
 		{
-			name:    "missing directory arg",
-			args:    []string{"ketch", "foo-bar"},
+			name:    "missing arg",
+			args:    []string{"ketch"},
 			wantErr: true,
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			os.Args = tt.args
```

Comment:
Just curious, why are you setting args like this as opposed to cmd.SetArgs?

Reply:
Good call. I made the switch.